kuma/static/js/interactive.js

```diff
@@ -7,6 +7,84 @@
         window.mdn.interactiveEditor.editorUrl ||
         'https://interactive-examples.mdn.mozilla.net';
 
+    /**
+     * Returns the interactive examples iframe performance entry
+     * @param {Array} perfEntries - The array of performance entries
+     */
+    function getInteractiveExamplesPerfEntry(perfEntries) {
+        for (var i = 0, l = perfEntries.length; i < l; i++) {
+            var currentEntry = perfEntries[i];
+            if (
+                currentEntry.initiatorType &&
+                currentEntry.initiatorType === 'iframe' &&
+                currentEntry.name.indexOf(
+                    'interactive-examples.mdn.mozilla.net'
```

Comment:
I'm not thrilled with hard-coding the production URL, since it makes local testing impossible, but I'm no closer to a solution than I was in September https://github.com/mdn/interactive-examples/issues/257